app/main.py

```diff
@@ -1,6 +1,29 @@
+import os
+from time import sleep
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+API_KEY = os.getenv("API_KEY")
+URL = "http://api.weatherapi.com/v1/current.json?"
+CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(URL + f"key={API_KEY}&q={CITY}")
```

Comment:
Use params keyword argument to pass query arguments. Do not hardcode params in url